HW0/P4/.ipynb_checkpoints/CS 205 HW0 P4-checkpoint.ipynb

```diff
@@ -0,0 +1,256 @@
nn76adx///14/PHH4+qek1AFkoEvifDRuHFj7N+/P+YMOjVB\njg9pVlaWX+Nlf0SRYAbExhMoK3Q6nbwVnd1ux5gxY9C6dWucOXMGv/76K9599100bdo0xqtNIk6R\nDHxJhI/GjRtj3759/CT2JCojGB/SQMQZCnp6vR4pKSmqK2uGA1ZzaLFYoNFoYLVasWzZMmzZsgXH\njh3DuXPn0LRpU9x4442YPHlyRMqcDz/8MDZs2IDatWsnx5JVPSQDXxLh48orr0RmZiZ0Oh0ee+wx\nPProo7FeUlwgWB/SsrIyAJcF9tEkzkQaQmcZ0hx6PB4sXrwYn332GRYuXIjrrrsODocDBw8eRH5+\nPoYMGRKRA9euXbuQlpaGoUOHJgNf1UMy8CURPmiG4fnz59GtWzfMnTsXnTp1ivWy4g7+fEhr1KiB\ndevWYerUqbj33nsBICrEmWjA6/XCZrMBgI8Z+PHjxzFq1Ch07NgRzz//fNTLuclB1FUWycCXhLKY\nMmUK0tLS8Mwzz8R6KVUC586dw/Dhw7Fv3z706dMHhw8flvQhVZo4E2n4GxL7wQcf4MMPP8ScOXMi\nMj1BDpKBr8oi6dWZRHiw2WzweDxIT0+H1WrFF198gZdeeinWy6oyeO2119C8eXN8/PHHsFgsQfmQ\nAvCxXlOTCTU7C5D1Dz1z5gxGjx6NFi1aYPv27TCbzVFfWxKJi2TGl4Qs/PHHH+jfvz+ACmr9kCFD\nkgJ7BSFHZM/6kObl5WHfvn2w2Wxo1qwZHwybNWsGrVYrKh8Q9gojKQ+QyvI4jsOaNWuwYMECzJo1\nCx07dox5dprM+KoskqXOJJKoinC73fj555954szhw4f9+pCyZVJ2gK+SxBka+cQOiQWACxcuYOzY\nsahduzZmzpyJ9PT0sF9LCSQDX5VFMvAlUfUgRkW/dOkS7r33Xpw4cQKNGjXC6tWrkZWVFeOVRg/B\n+pAqTZyRGhK7YcMGvP766/i///s/3H777THP8giDBw/Gzp07cfHiRdSuXRuvvPIKHnrooVgvKwll\nkAx8SVQ9iFHRJ0yYgJo1a2LChAmYOXMmCgsL+aG+iYpgfEiFIntAnu8mWbwJs7zi4mJMnDgRADBn\nzhxUq1YtSnedRBLJwJdEFYWwTNW8eXPs3LkTderUwdmzZ3HzzTfj8OHDMV6luhCMDykgPr2dDYQe\njwd2u71Slrdjxw68/PLLmDRpEvr376+aLC+JhEEy8CVRNSEMfNWqVUNhYSGAig2+evXq/J+TkEYo\nPqQOhwO0f+h0OuzevRsulwvXX3893nrrLVy8eBELFixArVq1Ynx3SSQoknKGJBIP8SLsVgM0Gg0a\nNmyIhg0b4u677wZQ0a/74YcfsGfPHsyaNcvHh7RatWqYO3cuFixYgM6dO8Pr9eLUqVNYsWIFvv/+\ne2RmZuK2227D6tWr0aVLF1x33XURW/vmzZv5cV7Dhw/nS6tJJCGFZOBLokqBSpx169bFmTNnULt2\n7VgvKW5hMBjQtm1btG3bFiNGjADHcTh9+jRGjBiB7du3o1u3bpg6dSqaNm2K1q1b46effkKNGjVw\n+PBhlJSU8GXUv/9/e3cP0lgWQAH4PNRCRFgQTCTJoiCi+SkCEZsRDBI1KEEQxE6MAREsXBtjp1bu\ngoh/iIVFbMSxCMYmYIo4VUxhwoIoWuRBCJLBBQsrJWSL7L4Zx5nJMObNNb7zgaDXwmN1Xu67Px8/\nqlZ8uVwOU1NTiEQiMBgMaG9vh8fj4QXO9F0sPnpTPB4PAoEAZmdnEQgEMDg4KDrSmyFJEubm5lBd\nXY1UKoW6ujrlHNJwOIyamhqEQiFlf6DVaoXP51M1UzweR3NzMxobGwEAIyMjODw8ZPHRd7H4qGz9\nvxT99vYWJpMJi4uL8Pv9GB4exs7OjrKdgUpna2tLOS0GKLzbM5vNMJvNQvJkMpkntzYYjUacnp4K\nyULlg8VHZWtvb++r45FI5Bcn0Y7PS+814Dtc+hm80phIZV6vFzqdDjabTRmbn5+H0WiE3W6H3W5H\nOBwWmLB8GQwGpNNp5ed0Oq3sSST6FhYfkcrGxsaeFZskSZiZmUEikUAikUBfX5+gdOXN4XDg+voa\nsizj4eEB+/v78Hg8omPRK8epTiKVdXZ2QpblZ+NF9tDSD6isrMTGxgZ6e3uRy+UwPj7OhS1UFIuP\nSJD19XXs7u7C4XBgeXlZU2eKlpLb7Ybb7RYdg8oIpzqJBJicnEQqlUIymURDQwMv9CX6hVh8RALU\n19crJ8v4fD7E43HRkYg0g8VHJMDNzY3yfTAYfLLiU4sODg5gsVhQUVGBs7Mz0XHojeM7PiKVfbnR\nfmFhAdFoFMlkEpIkoampCdvb26JjCmWz2RAMBjExMSE6CmkAi49IZV/baO/1egUkeb1aW1tFRyAN\n4VQnESGdTsPpdMJiscBqtWJtbQ1A4UZ7l8uFlpYW9PT04O7uTnBSopdj8RERqqqqsLKygvPzc8Ri\nMWxubuLi4gJLS0twuVy4urpCd3f3i26zd7lcsNlsz76Ojo5K+J8QFcepTiKCXq+HXq8HUDiPs62t\nDZlMBqFQCCcnJwCA0dFRdHV1/XT5HR8flywv0UvwEx8RPSHLMhKJBDo6OpDNZqHT6QAU7jrMZrOq\n/32eaENqY/ERkeL+/h5DQ0NYXV1FbW3tk9+peaN9MBiEyWRCLBZDf38/T2IhVUlFnq746EWkEY+P\njxgYGIDb7cb09DSAwmrLaDSq3GjvdDpxeXkpOCnRD/nmU1qx4iMiDZAKH+UCAP7J5/N/fDb+139j\nf0qS5AfwWz6f94vKSVQKLD4igiRJ7wB8APA3Ps30zAGIA3gP4HcAMoDhfD7PPQ1U1lh8RESkKVzc\nQkREmsLiIyIiTWHxERGRprD4iIhIU/4FN5+jjRm+RfoAAAAASUVORK5CYII=\n",
```

Comment:
dekatzenel,  you can click on "showing 13 changed file" and then directly click on the .ipynb file to open it with your ipython notebook.

fannyheneine,  it seems that you worked on the ipython notebook and then copy/paste on the python script. Once you deleted the .ipynb git saved the lastest checkpoint in a hidden folder .ipynb_checkpoints. You can see this folder with a "ls -a". Maybe next time you would want to delete this folder before pushing to your remote repo to avoid the multiple files. PS: I did the same thing